frontend/src/components/SourcesTable/__tests__/SourcesTable.spec.tsx

```diff
@@ -37,23 +37,23 @@ afterEach(cleanup);
 
 test('renders correctly with default props', () => {
   const renderer = TestRenderer.create(
-    <SourcesTable activeSource={source} sources={sourcesList} onRowClick={jest.fn()} />,
+    <SourcesTable activeSource={source} sources={sourcesList} />,
```

Comment:
Sources Table only has 2 props(activeSource and sources), since I got rid of onRowClick when i used context..so the other 2 are tested via SourcesTableRow